packages/astro/src/core/errors/errors-data.ts

```diff
@@ -523,7 +523,7 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 	/**
 	 * @docs
 	 * @see
-	 * - [Frontmatter injection](https://docs.astro.build/en/guides/markdown-content/#example-injecting-frontmatter)
+	 * - [Frontmatter injection](https://docs.astro.build/en/guides/markdown-content/#modifying-frontmatter-programmatically)
```

Comment:
```suggestion
	 * - [Modifying frontmatter programmatically](https://docs.astro.build/en/guides/markdown-content/#modifying-frontmatter-programmatically)
```

In other errors I made it so the link name is the header title, thoughts on doing the same here?

Reply:
Yes, that does make sense 👍